### PR TITLE
suggestions

### PR DESCRIPTION
**Pull request by salatine**
- wip suggestions
- fix optional fields and fix some typing bugs

### CODE REVIEW

File: vinylizer/Product.py [hunk 1]
```diff
@@ -12,10 +12,10 @@ class Product:
...
     is_national: bool
     pictures: list[str]
-    song_quantity: int
-    album_duration: float
-    release_year: int
-    label: str
+    song_quantity: int | None
+    album_duration: float | None
+    release_year: int | None
+    label: str | None
```
Comment (ekisu) [inline vinylizer/Product.py]: optional pfvr

File: vinylizer/__main__.py [hunk 2]
```diff
@@ -18,69 +19,93 @@ def main():
...
+            # campos opcionais
+            song_quantity = suggestion.song_quantity or 1,
+            album_duration = suggestion.album_duration or 0,
+            release_year = suggestion.release_year or None,
+            label = suggestion.label or None,
         )
         products.append(product)
-
-        if input('deseja continuar? [s]: ').lower() == 'n':
+        if input('deseja continuar? (s/n) [s]: ').lower() == 'n':
```
Comment (ekisu) [inline vinylizer/__main__.py]:
```suggestion
        if input('deseja continuar? [S/n]: ').lower() == 'n':
```

File: vinylizer/Product.py (hunk 1 above)
Comment (salatine) [reply vinylizer/Product.py]: ok

File: vinylizer/__main__.py (hunk 2 above)
Comment (salatine) [reply vinylizer/__main__.py]: okay

**Review by ekisu — APPROVED** [review]

**Pull request merged**